roottest/root/io/alloc/Makefile

```diff
@@ -1,93 +0,0 @@
-# the name should be changed accordingly in this list.
-
-TEST_TARGETS += customConstructor customConstructorACLiC
-
-# Search for Rules.mk in roottest/scripts
-# Algorithm:  Find the current working directory and remove everything after
-#  '*roottest/'.  Append the path for Rules.mk from within roottest, which 
-#  should be 'scripts/Rules.mk'.  The roottest path is saved in the
-#  ROOTTEST_HOME variable for use by the SUBDIRECTORIES variable and is 
-#  exported to eliminate the overhead of findding the path again during 
-#  recursive calls of gmake.
-# Since all makefiles should be under roottest or one of its
-#  subdirectories and all recursions of gmake are called by 
-#    'cd [DIR]; gmake ...'
-#  this algorithm should not fail in finding /roottest/ in the
-#  current working directory.  
-# Issues:  This algorithm will fail if a makefile is called from outside the 
-#  roottest folder, as in executing 'gmake -f ~/roottest/Makefile' from 
-#  the home directory.
-
-ifeq ($(strip $(ROOTTEST_HOME)),)
-   export ROOTTEST_HOME := $(shell git rev-parse --show-toplevel)/roottest/
-   ifeq ($(strip $(ROOTTEST_HOME)),)
-      export ROOTTEST_HOME := $(shell expr $(CURDIR) : '\(.*/roottest/\)')
-   endif
-   ifeq ($(strip $(ROOTTEST_HOME)),)
-      $(error The head of roottest was not found.  Set ROOTTEST_HOME)
-   endif
-endif
-
-include $(ROOTTEST_HOME)/scripts/Rules.mk
-
-
-# The name of this target should be changed according to the test being run.  Any name changes
-#  should also be reflected in the TEST_TARGETS variable above.
-
-mytest:
-	$(CMDECHO) echo mytest > /dev/null
-
-
-# Outputs a message if the FAIL variable is null
-
-testWithFailure:
-ifeq ($(FAIL),)
-	$(WarnFailTest)
-endif
-
-
-# Remember that sometest.log can be made automatically from
-#  from runsometest.C
-# This target attempts to create a log file (which should be the output of running the test), and
-#  compares its output with a prebuilt reference file.  The reference file should be copied from
-#  the log files produced during what is considered a successful run of the test.  The testWithDiff
-#  files should be renamed according to the test being run.
-# By default the TestDiff variable is set to compare the .log and .ref files
-# To run a diff and ignore whitespacing use $(TestDiffW)
-
-testWithDiff: testWithDiff.log testWithDiff.ref
-	$(TestDiff)
-
-customConstructorDict.cc : customConstructor.C linkdef.h $(ROOTCINT)
-	$(CMDECHO) rootcint -f customConstructorDict.cc customConstructor.C linkdef.h
-
-customConstructorDict.$(ObjSuf): customConstructorDict.cc 
-
-customConstructor.$(ObjSuf): customConstructor.C
-
-customConstructor.$(DllSuf): customConstructor.$(ObjSuf) customConstructorDict.$(ObjSuf)
-	$(BuildFromObjs)
-
-customConstructor.log: runcustomConstructor.C customConstructor.$(DllSuf)
-	$(CMDECHO) $(CALLROOTEXE) -b -l -q $< 2>&1 | sed -e 's/P13OTHER_ROOT_IO/class OTHER_ROOT_IO */' > $@  || handleError.sh --result=$$? --log=$@ --test=$<
```

Comment:
Do you know why we don't need this `sed` command anymore?

Reply:
We were missing (and/or I did not find it) a record of the backward incompatibility being tracked.

Reply:
Yes, as of the last time the alternate was run the output was platform dependent (sometimes the class name is printed mangled).